File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
      num_set = set(nums)
      lcs = 0
      for n in num_set:
        if (n-1) not in num_set:
            length = 1
            while (n+length) in num_set:
                length += 1
            lcs = max(length, lcs)
      return lcs
      

File: Data Structures & Algorithms/maximum-subarray/submission-0.cpp
class Solution {
public:
    int maxSubArray(vector<int>& nums) {
        int currMax = nums[0];
        int maxHere = nums[0];

        for(int i = 1; i < nums.size(); i++)
        {
            maxHere = max(nums[i], maxHere + nums[i]);
            currMax = max(currMax, maxHere);
        }
        return currMax;
    }
};
